src/App.js

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import Header from './components/Header';
+import PokemonTiles from './components/PokemonTiles';
+import apiRequest from "./RestApiRequest";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [data, setData] = useState( [] );
```

Comment:
`data` is nooit van het type array, dus dan zou ik de default waarde dan ook geen array maken, maar `undefined`:
```suggestion
    const [data, setData] = useState();
```